PlayTools/Controls/ControlMode.swift

```diff
@@ -55,6 +55,9 @@ public class ControlMode {
             }
             Toucher.writeLog(logMessage: "cursor show switched to \(show)")
             visible = show
+            if PlaySettings.shared.noKMOnInput {
```

Comment:
```suggestion
            if !PlaySettings.shared.noKMOnInput {
```

I'm generally against naming boolean variables as `not...` or `no...`, as it could become confusing when you want to flip this boolean variable. 

And this `show()` function here is not to switch editor mode (as described in your commit msg). This is called when `option` is pressed. Adding this thing here causes keymapping to switch as long as cursor hides or shows.

Yes this is the expected behaviour (and it does change as editor mode switches). Just to make sure no misunderstanding here :)

Reply:
> I'm generally against naming boolean variables as not... or no..., as it could become confusing when you want to flip this boolean variable.
Main issue being that I expect this to be a temporary thing. After we make it generally available we can remove it if it doesn't cause too many issues (like with root-as-workdir)

Reply:
Yes that's just a general principle, for this case specifically it's not a big problem. But I mean there should be a `!` before it, to only enable it if this new feature is disabled.